ci-operator/config/netobserv/flowlogs-pipeline/netobserv-flowlogs-pipeline-main.yaml

```diff
@@ -18,14 +34,32 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.16"
+      version: "4.18"
 resources:
   '*':
     limits:
       memory: 4Gi
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: qe-e2e-tests
+  optional: true
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      PATCH_FLOWLOGS_IMAGE: pipeline:flowlogs-pipeline
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      TEST_FILTERS: NonPreRelease;Serial;Disruptive;~59746&
```

Comment:
what's this cryptic "~59746&" ?

Reply:
Passing that to skip upgrade test. `59746` is the upgrade test-case number and `~<TEST-CASE>&` is added to skip a test